documentcloud.php

```diff
@@ -181,7 +181,7 @@ function prepare_oembed_fetch( $provider, $url, $args ) {
 		// Merge actual args with default attributes so that defaults are always
 		// sent to oEmbed endpoint
 		$default_atts = $this->get_default_atts();
-		$atts         = array_merge( $default_atts, $args );
+		$atts         = wp_parse_args( $default_atts, $args );
```

Comment:
Excellent, thanks @whyisjake. Seems like the [param order needs to be reversed though](https://codex.wordpress.org/Function_Reference/wp_parse_args#Usage)?